Objects/funcobject.c

```diff
@@ -423,6 +423,30 @@ func_get_annotations(PyFunctionObject *op, void *Py_UNUSED(ignored))
         op->func_annotations = PyDict_New();
         if (op->func_annotations == NULL)
             return NULL;
+
+    }
+    if (PyTuple_CheckExact(op->func_annotations)) {
+        PyObject *anns = op->func_annotations;
+        op->func_annotations = PyDict_New();
+        if (op->func_annotations == NULL) {
+            Py_DECREF(anns);
+            return NULL;
+        }
+
+        assert(PyTuple_Size(anns) % 2 == 0);
+
+        for (Py_ssize_t i = 0; i < PyTuple_Size(anns); i += 2) {
```

Comment:
Please use the macro `PyTuple_GET_SIZE()` instead of `PyTuple_Size()`, as `anns` is guaranteed to be a tuple here.

Reply:
Thanks, didn't notice such difference between `PyTuple_GET_SIZE()` and `PyTuple_Size()`